base_view_inheritance_extension/models/res_users.py

```diff
@@ -28,9 +29,9 @@ def write(self, vals):
         active = vals.get('active')
         if active is not None:
             self.env['ir.ui.view'].sudo().with_context(
-                active_test=False).search([
-                    ('user_ids', 'in', self.ids)]).with_context(
-                        active_test=True).filtered(
-                            lambda x: not x.user_ids - self).write({
-                                'active': active})
+                active_test=False,
+            ).search([
+                ('user_ids', 'in', self.ids)]).filtered(
+                    lambda x: not x.user_ids - self,
+                ).write({'active': active})
```

Comment:
If I'm not mistaken, due to `active_test=False`, `x.user_ids` includes inactive users. Therefore, the view active status is not adapted if there are other users on the view and they are all inactive. Maybe add a test to cover this case.

Reply:
The test case already exists when the `view5.active` is tested, and my wrong move of removing this `with_context` would have been caught had the tests run. But it turns out that they did/do not

Reply:
you should always run your tests client side before submitting something to reviewers, that's a basic courtesy.
OCA's Travis only picks up PRs to OCA's branches, as this is my clone, there's no travis